### DISCUSSION TITLE
Confusion of minor and patch update?

### DISCUSSION THREAD

**Discussion by georgkrause** [renovatebot/renovate#13669] [Request Help]
Hi again,

I got another question. I am really wondering what is happening here, I think this screenshot illustrates it:

![screenshot](https://user-images.githubusercontent.com/939889/150318101-4eceaaf4-8f7e-438a-8fdd-c243b38c77df.jpg)

From my point of view its a `minor` update, while Renovate considers this to be a patch. Why is this happening? Is it a bug or expected? How can I get around this?

**Comment by rarkins** [comment]
Do you have a lock file? And if so then what is the associated change of this dependency in the lock file? e.g. is it from `1.6.0` to `1.6.1`?

**Reply by georgkrause** [reply]
Yes, thats the case. So this is mostly happening because of the onboarding, good to know! Thank you a lot :)

**Reply by rarkins** [reply]
Yes, and I'm assuming you're using rangeStrategy=bump? You shouldn't get this mismatch once you're lock file is aligned with the range minimum in the package.json
